packages/core/parcel/src/cli.js

```diff
@@ -219,6 +223,7 @@ async function normalizeOptions(command): Promise<InitialParcelOptions> {
     serve,
     targets: command.target.length > 0 ? command.target : null,
     autoinstall: command.autoinstall !== false,
-    logLevel: command.logLevel
+    logLevel: command.logLevel,
+    workerFarm
```

Comment:
isn't there a default worker farm? does the CLI need to make one?

Reply:
it doesn't -- we create one if it's not passed. I'll remove it here.